Softeq.XToolkit.Permissions/PluginPermissionStatusExtensions.cs

```diff
@@ -1,27 +0,0 @@
-// Developed by Softeq Development Corporation
-// http://www.softeq.com
-
-using System.ComponentModel;
-using PluginPermissionStatus = Xamarin.Essentials.PermissionStatus;
-
-namespace Softeq.XToolkit.Permissions
-{
-    public static class PluginPermissionStatusExtensions
-    {
-        public static PermissionStatus ToPermissionStatus(this PluginPermissionStatus permissionStatus)
-        {
-            return permissionStatus switch
-            {
-                PluginPermissionStatus.Denied => PermissionStatus.Denied,
-                PluginPermissionStatus.Disabled => PermissionStatus.Denied,
-                PluginPermissionStatus.Granted => PermissionStatus.Granted,
-                PluginPermissionStatus.Restricted => PermissionStatus.Denied,
```

Comment:
@KirylAkulich1 I guess we can just add `PluginPermissionStatus.Restricted => PermissionStatus.Restricted`. What do you think?

Reply:
Agree. Fixed